### DISCUSSION TITLE
How can I set my page switch to dark theme anytime when users reload or re-open my home page?

### DISCUSSION THREAD

**Discussion by DoHaiSon** [alshedivat/al-folio#2808] [Q&A] [answered]
Dear all,
I custom some colors and features of my page only look good when it's in dark mode. So how can I set my page switch to dark theme anytime when users reload or re-open my home page? I don't want to remove the light theme but just a option and do not store this option in cache for the next access times.

Thx!!!

**Comment by george-gca** [comment, marked as answer]
The line responsible for storing this information is this:

https://github.com/alshedivat/al-folio/blob/89f7ca7b930ab8572c978d2537a478963e2c9389/assets/js/theme.js#L17

Now, I don't think removing it is a good choice, because probably every new page you open might always load the dark theme even though the light one was selected. I believe the best for you is to set the default as the dark theme instead of getting the system preference from the user (but only on the first load):

https://github.com/alshedivat/al-folio/blob/89f7ca7b930ab8572c978d2537a478963e2c9389/assets/js/theme.js#L213

or even disable light theme.

**Reply by DoHaiSon** [reply]
Oh thank u so much, I think the second option that is the best choice for me.
